Update index.md (#5387)
Kong/docs.konghq.com: app/_src/gateway/get-started/index.md

   You should receive a large JSON response with {{site.base_gateway}} configuration information.

1. Access Kong Manager
   

>     While the rest of this guide demonstrates configuring {{site.base_gateway}} via its Admin API, you can also use the Kong Manager UI to manage your Services, Routes, Plugins, and more. To access Kong Manager, navigate to the following URL in your browser: [http://localhost:8002](http://localhost:8002)

Reviewer: ```suggestion
    While the rest of this guide demonstrates configuring {{site.base_gateway}} using the Admin API, you can also use Kong Manager to manage your Services, Routes, Plugins, and more. To access Kong Manager, go to the following URL: [http://localhost:8002](http://localhost:8002)
```
